### PR TITLE
QOL Police Changes

### PR DESCRIPTION
**Pull request by Holidayy95**
Added Progress bars to 2 commands

Added a 'Fine' Command

### CODE REVIEW

File: client/job.lua [hunk 1]
```diff
@@ -349,13 +363,26 @@ RegisterNetEvent('police:client:CheckStatus', function()
             local player, distance = GetClosestPlayer()
             if player ~= -1 and distance < 5.0 then
                 local playerId = GetPlayerServerId(player)
-                QBCore.Functions.TriggerCallback('police:GetPlayerStatus', function(result)
-                    if result then
-                        for _, v in pairs(result) do
-                            QBCore.Functions.Notify(''..v..'')
+                QBCore.Functions.Progressbar("police_status", "Impounding Vehicle..", 10000, false, true, {
```
Comment (FjamZoo) [inline client/job.lua:366]: Why is this saying impoundinv vehicle when it should be checking status? Might just be a typo from copy paste,

File: server/main.lua [hunk 2]
```diff
@@ -187,6 +187,32 @@ QBCore.Commands.Add("pobject", Lang:t("commands.place_object"), {{name = "type",
...
 end)
 
+QBCore.Commands.Add("fine", "Fine Criminal Scum (Police Only)", {{name="id", help="Player ID"}, {name="amount", help="Fine Amount"}}, false, function(source, args)
+    local src = source
+    local Player = QBCore.Functions.GetPlayer(src)
+    local OtherPlayer = QBCore.Functions.GetPlayer(tonumber(args[1]))
+    local price = tonumber(args[2])
+
+    if Player.PlayerData.job.name == "police" or Player.PlayerData.job.name == "trooper" or Player.PlayerData.job.name == "bcso" or Player.PlayerData.job.name == "doc" and Player.PlayerData.job.onduty then
+        if OtherPlayer ~= nil then
```
Comment (FjamZoo) [inline server/main.lua:197]: You can chance 

`if OtherPlayer ~= nil then` to just `if OtherPlayer then`

**Review by FjamZoo — CHANGES_REQUESTED** [review]

**Comment by Holidayy95** [comment]
Also this goes with it, if you want the webhook

https://github.com/qbcore-framework/qb-smallresources/pull/234

**Comment by FjamZoo** [comment]
Hey, I saw that your server sided command for fining people does not have a distance check which means anyone can bill any player no matter of distance if you could update that to make it more secure, we would happily merge this into the Police Job

**Comment by FjamZoo** [comment]
Also instead of checking for multiple jobs server side please just use that way we dont need to add 100s of PoliceJobs

`PlayerData.job.type == "leo"`

**Pull request closed without merging**